Sming/SmingCore/Network/NetUtils.cpp

```diff
@@ -180,3 +180,35 @@ void NetUtils::debugPrintTcpList()
 	if(prt_none) debugf("LWIP_DEBUG: none\r\n");
 	return;
 }
+
+//search lowercase
+char* NetUtils::strstrl(char* source, const char* sequence)
```

Comment:
Why not just combine `strstr` and `tolower` ? You can see `Wiring/WString.cpp` for example usage.